zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/BasicAuthInterceptor.java

```diff
@@ -57,31 +46,6 @@ public static BasicAuthInterceptor create(
   @Override
   public HttpResponse execute(ClientRequestContext ctx, HttpRequest req) throws Exception {
     ctx.addAdditionalRequestHeader(HttpHeaderNames.AUTHORIZATION, basicCredentials);
-    return HttpResponse.from(delegate().execute(ctx, req)
```

Comment:
the responsibility here was misplaced. we don't need to duplicate response parsing